src/main/java/TaskList.java

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    ArrayList<Task> tasks;
+    Ui ui = new Ui();
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    public Task get(int i) {
+        return tasks.get(i);
+    }
+
+    public int size() {
+        return tasks.size();
+    }
+
+    /**
+     * Adds task to tasklist.
```

Comment:
Similar issue here with regards to the empty line between description and parameters section.